plugins/by-name/lsp-format/default.nix

```
@@ -91,28 +90,40 @@ lib.nixvim.plugins.mkNeovimPlugin {
 
   extraConfig = cfg: {
     warnings = lib.nixvim.mkWarnings "plugins.lsp-format" {
-      when = !config.plugins.lsp.enable;
+      when = !config.plugins.lspconfig.enable;
       message = ''
-        This plugin requires `plugins.lsp` to be enabled.
+        This plugin requires `plugins.lspconfig` to be enabled.
       '';
     };
 
-    plugins.lsp = {
-      onAttach =
-        lib.mkIf (cfg.lspServersToEnable == "all") # Lua
+    autoCmd = [
```

Should we have an autogroup?

```suggestion
    autoGroups.nixvim_lsp_fmt_setup.clear = false;

    autoCmd = [
```